pages/public_cloud/containers_orchestration/managed-rancher-service/getting-started/guide.en-gb.md

```diff
@@ -0,0 +1,276 @@
+  Provide the following parameters:
+  - the **Name** of the NodePool
+  - the OVHcloud Instance **Flavor** used by this NodePool
+  - the **Autoscaling** state
+  - the **Size** of your NodePool (number of nodes that will be created)  
+  - the **Monthly Billing** state (hourly by default)
+
+  Then click on 'Add Node Pool'. You can add multiple NodePools and then manage your list of Nodepools (note that the 'delete' action button of your first NodePool is grayed out till a second one is created).
+
+7. Click on **Finish & Create Cluster**
+
+8. From the Homepage your cluster is now in 'Provisioning' state
+![MKS Driver Cluster Configuration](images/mksdriver_provisioning.png)
+From the Cluser Management page, wait for your cluster to become 'Active'
+![MKS Driver Cluster Configuration](images/mksdriver_cluster_provisioning.png)
+![MKS Driver Cluster Configuration](images/mksdriver_cluster_provisioning_available.png)
+
+  Your cluster is now fully functional. You can click on 'Explore' to manage your MKS Cluster.
+
+
+##### Use OVHcloud as an **Infrastructure Provider**
+
+
+On this part we will detail how to use Rancher to create and manage Kubernetes clusters based on [OVHcloud Public Cloud Compute Instances](https://www.ovhcloud.com/en/public-cloud/compute/).
+
+  > [!warning]
+  >
+  > Deploying to OVHcloud will incur charges. For more information, refer to the [MKS](https://www.ovhcloud.com/en/public-cloud/prices/#568) and [Compute](https://www.ovhcloud.com/en/public-cloud/prices/) pricing pages.
+  >
+  >
+
+
+1. From the Rancher Homepage, click on 'Create'
+![Rancher Homepage](images/rancher-homepage.png)
+
+2. Use the OVHcloud 'Infrastructure Provider' by clicking on "OVHcloud Public Cloud" under the 'Provision new nodes and create using RKE2/k3s' section:
+![Cluster Creation](images/cluster-creation.png)
+
+3. Create your cloud credentials
+![Cluster Creation](images/PublicCloudDriver_creation_auth.png)
+Provide the following parameters:
+  - **Credential Name** (Optional)
+  - **Openstack Authentication URL**, default value can't be changed
+  - your Openstack **Username**, please refer to our Documentation for [Creating and deleting OpenStack users](https://help.ovhcloud.com/csm/en-public-cloud-compute-openstack-users?id=kb_article_view&sysparm_article=KB0050625)
+  - your Openstack **Password**, please refer to our Documentation for [Creating and deleting OpenStack users](https://help.ovhcloud.com/csm/en-public-cloud-compute-openstack-users?id=kb_article_view&sysparm_article=KB0050625)\
+
+  You have created the cloud credentials that will be used to provision nodes in your cluster. You can reuse these credentials for other node templates, or in other clusters.
+
+4. Click on **Get Project List** and select the **Project** were you want to create your cluster.\
+5. Click on **Continue**.
+
+6. Set your **Cluster Name** and **Cluster Decription**.\
+![Cluster Creation](images/PublicCloudDriver_creation_cloud_credential.png)
```

Comment:
The name of the image doesn't reflect the content

Reply:
```suggestion
![Public Cloud Driver - Cloud Credentials](images/PublicCloudDriver_creation_cloud_credential.png)
```